src/emulator.js

```diff
@@ -396,40 +395,25 @@ class Emulator {
     }
 
     parseRegisters(register) {
+        //Could convert this into an object of key/values to increase performance
         switch(register) {
-            case "a0":
-                return 0;
-            case "a1":
-                return 1;
-            case "a2":
-                return 2;
-            case "a3":
-                return 3;
-            case "a4":
-                return 4;
-            case "a5":
-                return 5;
-            case "a6":
-                return 6;
-            case "a7":
-            case "sp":
-                return 7;
-            case "d0":
-                return 8;
-            case "d1":
-                return 9; 
-            case "d2":
-                return 10; 
-            case "d3":
-                return 11; 
-            case "d4":
-                return 12; 
-            case "d5":
-                return 13; 
-            case "d6":
-                return 14; 
-            case "d7":
-                return 15;
+            case "a0": return 0;
+            case "a1": return 1;
+            case "a2": return 2;
+            case "a3": return 3;
+            case "a4": return 4;
+            case "a5": return 5;
+            case "a6": return 6;
+            case "a7": //TODO ??
```

Comment:
In the M68K specification register a7 and the sp (stack pointer) register are actually the same register in the hardware, basically an alias